proposals/drafts/ocdid_governance.rst

```diff
@@ -0,0 +1,113 @@
+
+Rationale
+=========
+
+Recently, a number of civic data organizations, tech companies, and US state governments have expressed interest in taking a greater role in the creation, upkeep, and wider adoption of OCDIDs. Currently, the governance structure for division IDs  as defined in OCDEP2 is "informal... led by the project’s early contributors and informed by the Open Civic Data Google Group." Increased adoption and interest makes this informality untenable for reasons including:
+
+* Bottlenecks arising when large amounts of new identifiers are proposed but only a few people have permission to review and approve pull requests.
+* Subject-matter experts and organizations that have extensive experience working with OCDIDs are willing to take greater responsibility, but currently do not have a formal path for doing so.
+* Government entities and other local experts who may wish to contribute to the OCDID repository do not have a clear idea of how the process for contirbution works, making it more difficult for them to justify contributing.
+
+We propose to formalize this structure in order to relieve some of the burden on those currently responsible for maintaining the repository, as well as allowing those whose work depends on OCDIDs to have more direct control on their upkeep.
+
+
+Implementation
+==============
+
+Roles & Responsibilities
+------------------------
+``User``
+~~~~~~~~
+Anyone using OCDIDs may contribute through Slack/Google group discussion, reporting issues, or assisting with subject-matter expertise.
+These channels for communication will be publicaly displayed on the Open Civic Data website.
+
+``Contributor``
+~~~~~~~~~~~~~~~
+Any individual or organization ``user`` may become a ``contributor`` by submitting a pull request adding, correcting, or aliasing jurisdictional OCDIDs.
+First time ``contributors`` will be advised to consult with an existing ``committer`` before submitting their first PR.
+
+``Committer`` 
+~~~~~~~~~~~~~
+``Committers`` have the ability to approve and commit pull requests from ``contributors`` and other ``committers``. 
+An initial cohort of ``committers`` will be determined by current project contributors.
+Any individual or organization who is a ``contributor`` and who agrees to the repsonsibilities listed here may apply to become a ``committer``. 
+
+* There will be a two month period where existing ``committers`` may raise objections to an applicant, otherwise the applying organization/individual will be considered approved.
+* In the case that an objection is raised, a discussion and vote will be had amongst existing ``committers``, with a simple majority necessary for approval.
+
+``Committers`` will be approved on an organizational basis, unless explicitly noted for an individual.
+
+* Multiple members of an organization may have committing ability, but the organization itself will be counted for all voting and approval processes.
```

Comment:
Will every individual with a commit bit need to be approved be current committers?

Reply:
GitHub operates on an individual level. It's unmanageable (and probably undesirable) to add an entire organization as committers (e.g. Google). If we are operating on an organizational basis, the organization should coordinate with committers which (small number of) people will be committers, and should notify committers when the individual leaves the organization… What's the (real) problems with individuals only? Bodies like the W3C have individuals on working groups representing organizations, but they are still individuals. The working groups don't have organizational membership.

Reply:
+1 to that, let's avoid 'organizations' - the one issue I see is that the delay in approval might hold an org up, but that could either be addressed through asking for an expedited decision or encouraging each org to have two committers

Reply:
My concerns were mostly around giving organizations the flexibility to designate/manage their own committers for efficiency's sake, and also ensuring that an organization with a large number of members couldn't wield outsized voting power. I'm fine going individual only and letting community norms/processes be a check.